content/docs/fairing/gcp-local-notebook.md

```diff
@@ -0,0 +1,228 @@
+weight = 30
++++
+
+This guide introduces you to using Kubeflow Fairing to train and deploy a
+model to Kubeflow on Google Kubernetes Engine, and Google Cloud ML Engine.
+As an example, this guide uses a local notebook to demonstrate how to:
+
+*  Train an XGBoost model in a local notebook,
+*  Use Kubeflow Fairing to train an XGBoost model remotely on Kubeflow,
+*  Use Kubeflow Fairing to train an XGBoost model remotely on Google Cloud ML Engine, 
+*  Use Kubeflow Fairing to deploy a trained model to Kubeflow, and
+*  Call the deployed endpoint for predictions.
+
+This guide has been tested on Linux and Mac OS X. Currently, this guide has not been
+tested on Windows.
+
+## Clone the Kubeflow Fairing Repository
+
+Clone the Kubeflow Fairing repository to download the files used in this example.
+
+```bash
+git clone https://github.com/kubeflow/fairing 
+cd fairing
+```
+
+## Set up Python, Jupyter Notebook, and Kubeflow Fairing
+
+1.  You need **Python 3.6** or later to use Kubeflow Fairing. To check if
+    you have Python 3.6 or later installed, run the following command:
+
+    ```bash
+    python3 -V
+    ```
+
+    The response should be something like this:
+
+    ```
+    Python 3.6.5
+    ```
+
+    If you do not have Python 3.6 or later, you can [download
+    Python](https://www.python.org/downloads/) from the Python Software
+    Foundation.
+
+1.  Use virtualenv to create a virtual environment to install Kubeflow
+    Fairing in. To check if you have virtualenv installed, run the
+    following command: 
+
+    ```bash
+    which virtualenv
+    ```
+
+    The response should be something like this.
+
+    ```bash
+    /usr/bin/virtualenv
+    ```
+
+    If you do not have virtualenv, use pip3 to install virtualenv.
+
+    ```bash
+    pip3 install --upgrade virtualenv
+    ```
+
+    Create a new virtual environment, and activate it.
+
+    ```bash
+    virtualenv venv --python=python3
+    source venv/bin/activate
+    ```
+
+1.  Install Jupyter Notebook.
+
+    ```bash
+    pip3 install --upgrade jupyter
+    ```
+
+1.  Install Kubeflow Fairing from the cloned repository.
+
+    ```bash
+    pip3 install --upgrade .
+    ```
+
+1.  Install the Python dependencies for the XGBoost demo notebook.
+
+    ```bash
+    pip3 install -r examples/prediction/requirements.txt
+    ```
+
+## Install and configure the Google Cloud SDK
+
+In order to use Kubeflow Fairing to train or deploy to Kubeflow on Google
+Kubernetes Engine, or Google Cloud Machine Learning Engine, you must configure
+your development environment with access to Google Cloud Platform. 
+
+1.  If you do not have the Google Cloud SDK installed, [install the Google
+    Cloud SDK][gcloud-install].
+
+1.  Use `gcloud` to set a default project.
+
+    ```bash
+    export PROJECT_ID=<your-project-id>
+    gcloud config set project $PROJECT_ID
+    ```
+
+1.  Kubeflow Fairing needs a service account to make API calls to Google Cloud
```

Comment:
Just GCP is OK here, as we've already established the abbreviation.

Reply:
Done